plugins/reporters/spdx/src/main/kotlin/SpdxDocumentModelMapper.kt

```diff
@@ -157,7 +157,7 @@ internal object SpdxDocumentModelMapper : Logging {
             creationInfo = SpdxCreationInfo(
                 comment = params.creationInfoComment,
                 created = Instant.now().truncatedTo(ChronoUnit.SECONDS),
-                creators = listOf("${SpdxConstants.TOOL}$ORT_FULL_NAME - ${Environment.ORT_VERSION}"),
+                creators = listOf("${SpdxConstants.TOOL} $ORT_FULL_NAME - ${Environment.ORT_VERSION}"),
```

Comment:
commits-msg: 
- To me it's unclear why the constant is now more convenient to use. (Looking at the current usages). Can you explain?
- To me it's unclear why it's desirable to make the space optional. Please provide a rationale.